app/lib/three_scale/json_representer.rb

```diff
@@ -37,6 +37,7 @@ def wraps_collection(name)
       self.instance_eval do
         include Representable::JSON::Collection
         include ThreeScale::CollectionRepresenter::XMLCollection
+        include ThreeScale::CollectionRepresenter::OrderedCollection
```

Comment:
This will affect all the data collection responses of 3scale API, enforcing the record to be sorted ascending by ID.

Reply:
Which is good 😄 👍

Reply:
This will prevent any other ordering in the API controllers.
We would need to document this behaviour for PostgreSQL at least

Reply:
Thinking. Probably the best is to order the relation/scope. Where we should touch it is the controller because once objects are loaded, you might even experience strange behaviour.

Imagine you have 1000 records. You want to load 100 only. They are picked randomly.
Ordering the loaded objeccts by `id` will not help you in this case because you might end up with the `1` and `999` in the same batch. Still `999` will be the last one but you would expect a range `1..100`